modules/rollback-operator-upgrades-cli.adoc

```diff
@@ -10,157 +11,80 @@ ifeval::["{context}" == "upgrade-cloudsvc-operator"]
+If you use Kubernetes, enter `kubectl` instead of `oc`.
 ====
-.. Ensure the Central deployment is scaled down by running the following command:
-** For {ocp}, run the following command:
+
+.. To delete the {olm} subscription, run the following command:
 +
 [source,terminal]
 ----
-$ oc scale -n <central namespace> –replicas=0 deploy/central
+$ oc -n rhacs-operator delete subscription rhacs-operator
 ----
-** For Kubernetes, run the following command:
 +
-[source,terminal]
-----
-$ kubectl scale -n <central namespace> –replicas=0 deploy/central
-----
-.. Save the following pod spec as a YAML file:
-+
-
-[source,yaml,subs="attributes+"]
-----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: get-previous-db-version
-spec:
-  containers:
-  - name: get-previous-db-version
-    image: registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:<rollback version>
-    command:
-    - sh
-    args:
-    - '-c'
-    - "cat /var/lib/stackrox/.previous/migration_version.yaml | grep '^image:' | cut -f 2 -d : | tr -d ' '"
-    volumeMounts:
-    - name: stackrox-db
-      mountPath: /var/lib/stackrox
-  volumes:
-  - name: stackrox-db
-    persistentVolumeClaim:
-      claimName: stackrox-db
-----
-.. Create a pod in your Central namespace by running the following command using the YAML file that you saved:
-** For {ocp}, run the following command:
+.Example output
 +
 [source,terminal]
 ----
-$ oc create -n <central namespace> -f pod.yaml
+subscription.operators.coreos.com "rhacs-operator" deleted
 ----
-** For Kubernetes, run the following command:
+
+.. To delete the CSV, run the following command:
 +
 [source,terminal]
 ----
-$ kubectl create -n <central namespace> -f pod.yaml
+$ oc -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
 ----
-.. After pod creation is complete, get the version by running the following command:
-** For {ocp}, run the following command:
 +
-[source,terminal]
-----
-$ oc logs -n <central namespace> get-previous-db-version
-----
-** For Kubernetes, run the following command:
+.Example output
 +
 [source,terminal]
 ----
-$ kubectl logs -n <central namespace> get-previous-db-version
+clusterserviceversion.operators.coreos.com "rhacs-operator.v4.8.4" deleted
 ----
 
-. Edit the `central-config.yaml` `ConfigMap` to set the `maintenance.forceRollBackVersion:<version>` parameter by running the following command:
-* For {ocp}, run the following command:
+ifndef::cloud-svc[]
+
+. Optional: To scale down the Central deployment, run the following command:
 +
-[source,terminal]
-----
-$ oc get configmap -n <central namespace> central-config -o yaml | sed -e "s/forceRollbackVersion: none/forceRollbackVersion: <version>/" | oc -n <central namespace> apply -f -
-----
-* For Kubernetes, run the following command:
+[NOTE]
+====
+Scale down the Central deployment only if you restore the database from a backup.
```

Comment:
This is just for external databases. I would remove this step entirely to avoid confusion.
It's also already mentioned in the appropriate section: https://99336--ocpdocs-pr.netlify.app/openshift-acs/latest/backup_and_restore/restore-acs#restoring-an-external-postgresql-database_restore-acs